hack/update-toc.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+# forked from github.com/ekalinin/github-markdown-toc/gh-md-toc
+wget -O ${OS_OUTPUT_BINPATH}/gh-md-toc https://raw.githubusercontent.com/damemi/github-markdown-toc/hide-footer/gh-md-toc
```

Comment:
Should the `gh-md-toc` script just be added to the descheduler repo instead of downloading it from another repo?

Reply:
What about to duplicate https://github.com/kubernetes-sigs/metrics-server/blob/master/Makefile#L177-L189 ? Looks quite shorter.

Reply:
Oh sweet, I did not even know there was a k8s toc generator already. I'll update to use that instead.

We can vendor it like we do with our other generators so it doesn't have to be downloaded each time.